PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/helpers/arm/ResourceTemplateView.java

```diff
@@ -66,15 +62,15 @@
     private JButton updateDeploymentButton;
     private JPanel contentPane;
     private JPanel editorPanel;
-    private JPanel editorPanels;
     private JPanel parameterPanel;
     private JLabel lblEditorPanel;
     private JLabel lblParametersPanel;
+    private JSplitPane armSplitPanel;
     private DeploymentNode node;
     private Project project;
     private static final String PROMPT_MESSAGE_SAVE_TEMPALTE = "Would you like to save the template file before you exit";
     private static final String PROMPT_MESSAGE_SAVE_PARAMETERS = "Would you like to save the parameters file before you exit";
-    private static final String PROMPT_MESSAGE_UPDATE_DEPLOYMENT = "Are you sure you want to update the deployment with the modified template";
+    private static final String PROMPT_MESSAGE_UPDATE_DEPLOYMENT = "Are you sure you want to update the deployment";
```

Comment:
Problematic statement, Would you like to update the deployment

Reply:
This is a confirm message so shall we change it to 'Are you sure to update the deployment'?

Reply:
Sure